config/deployer.sample.json

```diff
@@ -13,6 +13,10 @@
     "Version": 6.5,
     "CreateRole": false
   },
+  "JobServerSettings":{
+    "InstanceCount": 0,
+    "InstanceType": "t3a.micro"
```

Comment:
Are we on a tight budget? :grinning: 
I think we can safely default to what we use for common app server instance, `c5.xlarge`. Overall I think the less we vary on these the more consistent results we'll be getting.

Reply:
Ah, OK. Thought a small instance would be enough though 😄

Reply:
Yes, but let's not make that our problem. My argument is more about focusing on load-testing and not suggesting ideal values for production. It's easy enough to lower it if needed :)

Reply:
Haha .. `c5.xlarge` ftw!